index.html

```diff
@@ -0,0 +1,393 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,700;0,900;1,400&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+      integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="main.css"/>
+
+    <title>West Africa Farming</title>
+</head>
+
+<body>
+    <div class="container-fluid">
+        <!--upper navigation-->
+
+
+        <div class="row top_navigation">
```

Comment:
- [ ] I would like to recommend you wrap up inside the `<nav>`  tag [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element which it uses to represent a section of a page whose purpose is to provide navigation links, either within the current document or to other documents. Please fix it  for both the about and index pages navigation and make also your both navigation similar with the same content :+1: